vent/core/rq_worker/watch.py

```diff
@@ -122,13 +122,13 @@ def file_queue(path, template_path="/vent/"):
     import ast
     import docker
     import json
+    import netifaces
```

Comment:
this needs to be in the requirements file for `rq_worker` as well, since it's not in the standard library.